tests/rkt_fetch_test.go

```diff
@@ -161,7 +161,11 @@ func testFetchDefault(t *testing.T, arg string, image string, imageArgs string,
 	if err := expectWithOutput(child, remoteFetchMsg); err != nil {
 		t.Fatalf("%q should be found: %v", remoteFetchMsg, err)
 	}
-	child.Wait()
+	err := child.Wait()
+	status := getExitStatus(err)
+	if status != 0 {
+		t.Skip("remote fetching failed, probably a network failure. Skipping...")
```

Comment:
Also, can we have a more prominent message? I am concerned, that this will be just overlooked, in case we have a real problem with `fetch`.